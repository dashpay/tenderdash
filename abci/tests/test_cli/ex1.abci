echo hello
info
prepare_proposal 1 "abc"
process_proposal 1 "abc"
finalize_block 1 0000000000000000000000000000000000000000000000000000000000000000 "abc"
info
query "abc"
prepare_proposal 2 "def=xyz" "ghi=123"
process_proposal 2 "def=xyz" "ghi=123"
finalize_block 2 3C868EF29ED961094EE2C48BDD78CDC83C7AA426E763024985AF9F0F569EEFBE "def=xyz" "ghi=123"
query "def"
prepare_proposal 3 "preparedef"
process_proposal 3 "def"
process_proposal 3 "preparedef"
prepare_proposal 3
process_proposal 3
finalize_block 3 243BA0AC2622E2E7ACFEF366E85E503ACEBA1C950357436B554CA253492420AB
