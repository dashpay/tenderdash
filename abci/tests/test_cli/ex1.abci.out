> echo hello
-> code: OK
-> data: hello
-> data.hex: 0x68656C6C6F

> info
-> code: OK
-> data: {"appHash":"0000000000000000000000000000000000000000000000000000000000000000"}
-> data.hex: 0x7B2261707048617368223A2230303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030227D

> prepare_proposal 1 "abc"
-> code: OK
-> data: {"appHash":"0000000000000000000000000000000000000000000000000000000000000000"}
-> data.hex: 0x7B2261707048617368223A2230303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030227D
-> code: OK
-> log: Succeeded. Tx: abc action: UNMODIFIED

> process_proposal 1 "abc"
-> code: OK
-> data: {"appHash":"0000000000000000000000000000000000000000000000000000000000000000"}
-> data.hex: 0x7B2261707048617368223A2230303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030227D
-> status: ACCEPT

> finalize_block 1 0000000000000000000000000000000000000000000000000000000000000000 "abc"
-> code: OK
-> data.hex: 0x6576656E74733A3C747970653A2276616C5F757064617465732220617474726962757465733A3C6B65793A2273697A65222076616C75653A223022203E20617474726962757465733A3C6B65793A22686569676874222076616C75653A223122203E203E20

> info
-> code: OK
-> data: {"appHash":"0000000000000000000000000000000000000000000000000000000000000000"}
-> data.hex: 0x7B2261707048617368223A2230303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030303030227D

> query "abc"
-> code: OK
-> log: exists
-> height: 1
-> key: abc
-> key.hex: 616263
-> value: abc
-> value.hex: 616263

> prepare_proposal 2 "def=xyz" "ghi=123"
-> code: OK
-> data: {"appHash":"3C868EF29ED961094EE2C48BDD78CDC83C7AA426E763024985AF9F0F569EEFBE"}
-> data.hex: 0x7B2261707048617368223A2233433836384546323945443936313039344545324334384244443738434443383343374141343236453736333032343938354146394630463536394545464245227D
-> code: OK
-> log: Succeeded. Tx: def=xyz action: UNMODIFIED
-> code: OK
-> log: Succeeded. Tx: ghi=123 action: UNMODIFIED

> process_proposal 2 "def=xyz" "ghi=123"
-> code: OK
-> data: {"appHash":"3C868EF29ED961094EE2C48BDD78CDC83C7AA426E763024985AF9F0F569EEFBE"}
-> data.hex: 0x7B2261707048617368223A2233433836384546323945443936313039344545324334384244443738434443383343374141343236453736333032343938354146394630463536394545464245227D
-> status: ACCEPT

> finalize_block 2 3C868EF29ED961094EE2C48BDD78CDC83C7AA426E763024985AF9F0F569EEFBE "def=xyz" "ghi=123"
-> code: OK
-> data.hex: 0x6576656E74733A3C747970653A2276616C5F757064617465732220617474726962757465733A3C6B65793A2273697A65222076616C75653A223022203E20617474726962757465733A3C6B65793A22686569676874222076616C75653A223222203E203E20

> query "def"
-> code: OK
-> log: exists
-> height: 2
-> key: def
-> key.hex: 646566
-> value: xyz
-> value.hex: 78797A

> prepare_proposal 3 "preparedef"
-> code: OK
-> data: {"appHash":"300F6A9CA498936A45F61A6618F3E2610757FB7F2A7276A91564470EDA220516"}
-> data.hex: 0x7B2261707048617368223A2233303046364139434134393839333641343546363141363631384633453236313037353746423746324137323736413931353634343730454441323230353136227D
-> code: OK
-> log: Succeeded. Tx: def action: ADDED
-> code: OK
-> log: Succeeded. Tx: preparedef action: REMOVED

> process_proposal 3 "def"
-> code: OK
-> data: {"appHash":"300F6A9CA498936A45F61A6618F3E2610757FB7F2A7276A91564470EDA220516"}
-> data.hex: 0x7B2261707048617368223A2233303046364139434134393839333641343546363141363631384633453236313037353746423746324137323736413931353634343730454441323230353136227D
-> status: ACCEPT

> process_proposal 3 "preparedef"
-> code: OK
-> data: {"appHash":"300F6A9CA498936A45F61A6618F3E2610757FB7F2A7276A91564470EDA220516"}
-> data.hex: 0x7B2261707048617368223A2233303046364139434134393839333641343546363141363631384633453236313037353746423746324137323736413931353634343730454441323230353136227D
-> status: ACCEPT

> prepare_proposal 3
-> code: OK
-> data: {"appHash":"243BA0AC2622E2E7ACFEF366E85E503ACEBA1C950357436B554CA253492420AB"}
-> data.hex: 0x7B2261707048617368223A2232343342413041433236323245324537414346454633363645383545353033414345424131433935303335373433364235353443413235333439323432304142227D

> process_proposal 3
-> code: OK
-> data: {"appHash":"243BA0AC2622E2E7ACFEF366E85E503ACEBA1C950357436B554CA253492420AB"}
-> data.hex: 0x7B2261707048617368223A2232343342413041433236323245324537414346454633363645383545353033414345424131433935303335373433364235353443413235333439323432304142227D
-> status: ACCEPT

> finalize_block 3 243BA0AC2622E2E7ACFEF366E85E503ACEBA1C950357436B554CA253492420AB
-> code: OK
-> data.hex: 0x6576656E74733A3C747970653A2276616C5F757064617465732220617474726962757465733A3C6B65793A2273697A65222076616C75653A223022203E20617474726962757465733A3C6B65793A22686569676874222076616C75653A223322203E203E20

